src/components/MegadraftEditor.js

```diff
@@ -152,6 +154,23 @@ export default class MegadraftEditor extends Component {
     }
   }
 
+  getSidebarComponent = () => {
+    const { sidebarRendererFn, editorState } = this.props;
+    let SidebarComponent = null;
+    if(typeof sidebarRendererFn === "function") {
+      SidebarComponent = sidebarRendererFn(this.plugins, editorState);
```

Comment:
Yeah, that was great, thanks for the example!

My point is that with this solution we have to assume that the `sidebarRendererFn` is in a scope that has access to the same `onChange` method as the editor.

So, everyone using a custom sidebar needs to pass the `onChange` method both to the editor and to the sidebar.